scripts/lang_fill_missing.py

```
@@ -0,0 +1,184 @@
+            stats[path.name] = len(touched)
+            all_touched.extend((path.name, *t) for t in touched)
+            dirty.add(path)
+    return stats, all_touched, dirty
+
+
+def main() -> int:
+    ap = argparse.ArgumentParser(description=__doc__)
+    ap.add_argument("--dry-run", action="store_true",
+                    help="只打印变更，不写回文件")
+    ap.add_argument("--sources-only", action="store_true",
+                    help="只跑阶段 A（数据源 → lang）")
```

Ruff (0.16.1)

[warning] String contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF001)